### ISSUE TITLE
Unsupported target "distTag" using bun

### ISSUE THREAD

**Issue by Punie** [raineorshine/npm-check-updates#1469]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)

---

## Steps to Reproduce

Steps:

On a project with `bun.lockb` present run:

```
bunx ncu --target @next
```

## Current Behavior

Error:

```
> bunx ncu --target @next
Using bun
Upgrading /Users/dev/project/package.json
[======--------------] 13/46 28%
Unsupported target "distTag" using bun
Supported version targets are: latest, newest, greatest, minor, patch, semver, and tags (e.g. @next)
```

## Expected Behavior

I would expect it to be possible to use such tags as bun uses the npm registry just fine.
Is there any particular reason why distTags are not supported with bun?

**Comment by raineorshine** [comment]
Hi, thanks for reporting.

That seems to be an oversight on my part. Because `bun` uses the npm registry, npm-check-updates basically reuses all the logic from npm fetching, but distTags seems to have been omitted.

I added `distTags` and published to `v17.1.6`. Let me know if that works.

**Comment by Punie** [comment]
I just tried it with the new `v17.1.6` and it works perfectly 👍 

Thanks a lot for the quick response and fix ❤️
